scripts/sitecustomize.py

```
@@ -0,0 +1,35 @@
+
+def _maybe_start_coverage() -> None:
+    if not os.environ.get("COVERAGE_PROCESS_START"):
+        return
+    try:
+        import coverage
+    except ImportError:
+        return
+
+    try:
+        coverage.process_startup()
+    except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)